src/test/java/test/listeners/InterceptorInvokeTwiceSimulateListener.java

```diff
@@ -0,0 +1,49 @@
+package test.listeners;
+
+import java.util.List;
+import org.testng.IMethodInstance;
+import org.testng.IMethodInterceptor;
+import org.testng.ITestContext;
+import org.testng.ITestListener;
+import org.testng.ITestResult;
+
+/**
+ * @author github username: binwu-ericsson
```

Comment:
We don't use `@author` tag. git is already here to track author.

Reply:
aha, @author has been removed and javadoc was added.